### PR TITLE
Support for wtns calc and prover options

### PR DESCRIPTION
**Pull request by OBrezhniev**
Support passing witness calc & prover options:
* Pass `memorySize` option to witness calc
* Pass `singleTread` option to prover

Other updates:
* Update deps (including circom_runtime which supports memorySize option)
* Fix puppeteer tests on GitHub

### CODE REVIEW

File: src/wtns_debug.js [hunk 1]
```diff
@@ -34,10 +34,8 @@ export default async function wtnsDebug(_input, wasmFileName, wtnsFileName, symN
     const wasm = await fdWasm.read(fdWasm.totalSize);
     await fdWasm.close();
 
-
-    let wcOps = {
-        sanityCheck: true
-    };
+    let wcOps = options || {};
+    wcOps.sanityCheck = true;
```
Comment (Kolezhniuk) [inline src/wtns_debug.js:38]: `const wcOps = {... options, sanityCheck: true};`

**Review by Kolezhniuk — CHANGES_REQUESTED** [review]

File: src/wtns_debug.js [hunk 2]
```diff
@@ -65,7 +63,7 @@ export default async function wtnsDebug(_input, wasmFileName, wtnsFileName, symN
     wcOps.sym = sym;
 
     const wc = await WitnessCalculatorBuilder(wasm, wcOps);
-    const w = await wc.calculateWitness(input);
+    const w = await wc.calculateWitness(input, true);
```
[the remark this answers is not in the document]
Comment (OBrezhniev) [reply src/wtns_debug.js:66]: It sets sanityCheck param to true.

File: package.json [hunk 3]
```diff
@@ -59,7 +59,7 @@
     "@iden3/binfileutils": "0.0.12",
     "bfj": "^7.0.2",
     "blake2b-wasm": "^2.4.0",
-    "circom_runtime": "0.1.25",
+    "circom_runtime": "iden3/circom_runtime#6d4843929b61baba03be8090b094946e54717192",
```
[the remark this answers is not in the document]
Comment (OBrezhniev) [reply package.json:62]: @Kolezhniuk, 

Yes, please review iden3/circom_runtime#111

**Review by Kolezhniuk — DISMISSED** [review]

**Review by Kolezhniuk — DISMISSED** [review]

**Comment by rrrliu** [comment]
hey @Kolezhniuk @OBrezhniev 👋 checking in on this here, are we good to merge here?

this change would be super helpful for our app, which needs the memorySize option to ensure proving doesn't crash on mobile browsers :)

**Comment by OBrezhniev** [comment]
Yes, I hope a final review and release would happen next week.

**Review by Kolezhniuk — DISMISSED** [review]

**Review by Kolezhniuk — APPROVED** [review]

**Comment by rrrliu** [comment]
thanks for the review @Kolezhniuk -- excited for the release!

**Comment by OBrezhniev** [comment]
Hi @rrrliu , it's merged and released.

**Pull request merged**